RNBitmovinPlayer.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "12.4" }
+  s.platforms    = { :ios => "12.4", :tvos => "12.4" }
```

Comment:
Just noticed now, why do we require the `12.4` versions instead of `12.0`?
We support from 12.0 from the iOS SDK side.

Reply:
That's because RN 0.69 [requires](https://github.com/facebook/react-native/commit/982ca30de079d7e80bd0b50365d58b9048fb628f) at least iOS `12.4` to work and the example app is using RN 0.69 on dev right now.

I set this requirement on `.podspec` by the time I worked on the example app maybe, so it was my mistake. I think we can safely set these versions to `12.0` since only applications require `12.4` to work with RN 0.69.

Reply:
I see. Thanks.